cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaClusterTest.java

```diff
@@ -867,44 +867,6 @@ public void testLoadBalancerExternalTrafficPolicyLocalFromListener() {
         }
     }
 
-    @Test
-    public void testLoadBalancerExternalTrafficPolicyLocalFromTemplate() {
-        Kafka kafkaAssembly = new KafkaBuilder(ResourceUtils.createKafka(namespace, cluster, replicas,
-                image, healthDelay, healthTimeout, metricsCm, jmxMetricsConfig, configuration, emptyMap()))
-                .editSpec()
-                    .editKafka()
-                        .withNewListeners()
-                            .addNewGenericKafkaListener()
-                                .withName("external")
-                                .withPort(9094)
-                                .withType(KafkaListenerType.LOADBALANCER)
-                                .withTls(true)
-                            .endGenericKafkaListener()
-                        .endListeners()
-                        .withNewTemplate()
-                            .withNewExternalBootstrapService()
-                                .withExternalTrafficPolicy(ExternalTrafficPolicy.LOCAL)
-                            .endExternalBootstrapService()
-                            .withNewPerPodService()
-                                .withExternalTrafficPolicy(ExternalTrafficPolicy.LOCAL)
-                            .endPerPodService()
-                        .endTemplate()
-                    .endKafka()
-                .endSpec()
-                .build();
-        KafkaCluster kc = KafkaCluster.fromCrd(kafkaAssembly, VERSIONS);
-
-        // Check external bootstrap service
-        Service ext = kc.generateExternalBootstrapServices().get(0);
-        assertThat(ext.getSpec().getExternalTrafficPolicy(), is(ExternalTrafficPolicy.LOCAL.toValue()));
-
-        // Check per pod services
-        for (int i = 0; i < replicas; i++)  {
-            Service srv = kc.generateExternalServices(i).get(0);
-            assertThat(srv.getSpec().getExternalTrafficPolicy(), is(ExternalTrafficPolicy.LOCAL.toValue()));
-        }
-    }
-
     @Test
     public void testLoadBalancerExternalTrafficPolicyLocalConflict() {
```

Comment:
I think you can remove this test as well since the conflict does not exist anymore, or?

Reply:
I think this test is still needed as it's part of the `testLoadBalancerExternalTrafficPolicyLocalFromListener` and we still wanted to make sure the policy exists with the new configuration correct?